trace/internal/telemetry/id.go

```diff
@@ -50,7 +50,7 @@ func (sid SpanID) String() string {
 	return hex.EncodeToString(sid[:])
 }
 
-// IsEmpty returns true if the span ID contains at least one non-zero byte.
+// IsEmpty reports whether the SpanID contains only zero bytes.
```

Comment:
The previous doc what incorrect (it should be `false` instead of `true`).
I do not think we need a changelog entry for it.